ipaclient/plugins/vault.py

```diff
@@ -304,7 +279,8 @@ def forward(self, *args, **options):
                 password = password.rstrip('\n')
 
             else:
-                password = get_new_password()
+                password = self.api.Backend.textui.prompt_password(
```

Comment:
Is this LGTM meant for the whole patch or just this one-liner? If the prior, please, don't add your LGTMs to single lines, if the latter, please, do complete reviews.